presto-sqlserver/src/main/java/io/prestosql/plugin/sqlserver/SqlServerClient.java

```diff
@@ -233,6 +240,32 @@ public boolean isLimitGuaranteed(ConnectorSession session)
         return true;
     }
 
+    @Override
+    protected String createTableSql(RemoteTableName remoteTableName, List<String> columns, ConnectorTableMetadata tableMetadata)
+    {
+        return format("CREATE TABLE %s (%s) WITH (DATA_COMPRESSION = %s)", quoted(remoteTableName), join(", ", columns), getDataCompression(tableMetadata.getProperties()));
```

Comment:
This is correct because currently SQL Server does not seem to support enabling table compression by default for a schema or database. However, they can add in the future.
Therefore today NONE is a good default, but can become a wrong choice without a code change on our part.

Let's replace NONE with UNSET (or DEFAULT). Then do not add `WITH (DATA_COMPRESSION` clause when compression is the UNSET.

cc @kokosing

Reply:
Correct me if I am wrong, NONE means no compression.

> Let's replace NONE with UNSET (or DEFAULT). Then do not add WITH (DATA_COMPRESSION clause when compression is the UNSET.

Let's implement UNSET by not setting property value, so let's make it nullable and when it is null we should just use sql server system default behaviour.

Reply:
But when we run `SHOW CREATE TABLE` for any table from SQLServer we won't be able to capture the `UNSET`/`DEFAULT` rather we get only `NONE`. How can we handle in that situation ?